components/shared/carousel/RhCarousel.tsx

```diff
@@ -0,0 +1,52 @@
+import { css, StyleSheet } from "aphrodite";
+import { ReactElement, useState } from "react";
+import RhCarouselItem, { RhCarouselItemProps } from "./RhCarouselItem";
+
+type Props = {
+  rhCarouselItemProps: RhCarouselItemProps[];
+};
+
+const RhCarouselControl = ({
```

Comment:
I know you didn't put it there, but can we move the carousel outside the shared folder? I think everything is technically shared

Reply:
I'll do it at the finalizing PR. It will mess with the file paths if i do it in mid-node